service/cloudconfig/template.go

```diff
@@ -0,0 +1,33 @@
+package cloudconfig
+
+const (
+	getKeyVaultSecretsTemplate = `#!/bin/bash -e
+
+KEY_VAULT_HOST={{ .VaultName }}.vault.azure.net
+AUTH_TOKEN=$(curl http://localhost:50342/oauth2/token --data "resource=https://vault.azure.net" -H Metadata:true | jq -r .access_token)
```

Comment:
How can be sure `jq` is installed?

Reply:
Good point on the dependency. I tested on a CoreOS stable node and `jq` was pre-installed. Let me check on that.

Reply:
Checked on Core OS stable in westeurope and `jq` is pre installed.

```
Container Linux by CoreOS stable (1465.7.0)
jq --version
jq-1.5
```

We could install it anyway. Not sure. Using the operator binary would also fix this.